ui/hooks/useMultichainAccountsIntroModal.ts

```diff
@@ -1,12 +1,11 @@
 import { useState, useEffect } from 'react';
-import { lt as semverLt } from 'semver';
+import { lt as semverLt, parse as semverParse } from 'semver';
```

Comment:
All changes in this file come from this PR https://github.com/MetaMask/metamask-extension/pull/38382, which was not included in 13.12.0, but was cherry-picked in 13.11.1.